non-oss/commands/backup/client_helper.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module ClientHelper
+  # FIXME These kube client related helpers should be somewhere more common
+  def client
+    @client ||= create_client
+  end
+
+  def create_client
+    if ENV['KUBE_TOKEN'] && ENV['KUBE_CA'] && ENV['KUBE_SERVER']
```

Comment:
(We have this snippet several times in several projects, it should probably be in k8s-client)